RELEASE_NOTES.md

```diff
@@ -415,7 +415,7 @@ changes (where available).
 ## RawSpeed changes
 
 
-## Camera support, compared to 4.2
+## Camera support, compared to 4.4
```

Comment:
This section is auto generated when I create the release. But not a problem to tweak it now anyway.